docker/owlbot/nodejs_mono_repo/Dockerfile

```diff
@@ -50,5 +53,4 @@ RUN find /synthtool -type d -exec chmod a+x {} \;
 RUN cd /synthtool && mkdir node_modules && npm i gts@3.1.0 google-gax@3.5.2 \
     typescript@4.7.4 @google-cloud/typeless-sample-bot@1.0.3
 
-ENTRYPOINT [ "/bin/bash" ]
-CMD [ "/synthtool/docker/owlbot/nodejs_mono_repo/entrypoint.sh" ]
+ENTRYPOINT [ "/bin/bash", "/synthtool/docker/owlbot/nodejs_mono_repo/entrypoint.sh" ]
```

Comment:
curious what the reason for switching from `CMD` to `ENTRYPOINT` was?

Reply:
This is so that we can pass command-line arguments to the Docker container, i.e.,:

`docker run ${IMAGE} packages/google-cloud-compute,packages/google-cloud-asset`

See example: https://github.com/googleapis/repo-automation-bots/blob/main/packages/owlbot-bootstrapper/common-container/Dockerfile which then passes these args: https://github.com/googleapis/repo-automation-bots/blob/e3c22cecfb1495b36f50bf5d35aebedc3e2696d1/packages/owlbot-bootstrapper/cli/cloudbuild-owlbot-bootstrapper.yaml#L24